node/hyperbahn/handler.js

```diff
@@ -28,6 +28,9 @@ var util = require('util');
 var TChannelJSON = require('../as/json');
 var TChannelEndpointHandler = require('../endpoint-handler');
 
+var MAX_RELAY_AD_ATTEMPTS = 2;
```

Comment:
even if we define defaults here, can we then use them as an option / self property later on?